src/wp-admin/includes/upgrade.php

```diff
@@ -2396,7 +2399,45 @@ function upgrade_650() {
 		wp_set_option_autoload_values( $autoload );
 	}
 }
+/**
+ * Executes changes made in WordPress 6.5.0.
+ *
+ * @ignore
+ * @since 6.7.0
+ *
+ * @global int  $wp_current_db_version The old (current) database version.
+ */
+function upgrade_670() {
+	global $wp_current_db_version;
+
+	if ( $wp_current_db_version < 57160 ) {
+		$options_ids = array(
+			'auto_plugin_theme_update_emails',
+			'recently_activated',
+			'_wp_suggested_policy_text_has_changed',
+			'dashboard_widget_options',
+			'ftp_credentials',
+			'recently_edited',
+			'adminhash',
+			'nav_menu_options',
+			'active_plugins',
+			'uninstall_plugins',
+			'wp_force_deactivated_plugins',
+			'delete_blog_hash',
+			'allowedthemes',
+			'admin_email',
+			'recently_activated',
+			'https_detection_errors',
+			'fresh_site',
+			'upload_path',
+			'admin_email',
+			'admin_email_lifespan',
+		);
 
+		$autoload = array_fill_keys( $options_ids, 'no' );
+		wp_set_option_autoload_values( $autoload );
```

Comment:
```suggestion
		wp_set_options_autoload( $options_ids, 'no' );
```
The function will take care.